release/pypi/prep_binary_for_pypi.sh

```diff
@@ -88,10 +88,10 @@ for whl_file in "$@"; do
             record_file="${dirname_dist_info_folder}/${basename_dist_info_folder/${version_with_suffix}/${version_no_suffix}}/RECORD"
             if [[ -e $record_file ]]; then
                 echo "Generating new record file $record_file"
-                rm -f $record_file
+                : > "$record_file"
```

Comment:
Same question here, please educate. Also, is the permission change necessary from 100644 -> 100755?

Reply:
> is the permission change necessary from 100644 -> 100755?

Ah, I guess not. I changed the permission to include the `+x` (execute bit) so that I could run the script: `./release/pypi/prep_binary_for_pypi.sh` If this script was somehow being launched another way, I'm happy to revert the permissions to `100644`. Let me know.